mico-core/src/main/java/io/github/ust/mico/core/REST/ApplicationController.java

```diff
@@ -50,8 +47,10 @@
 
 @RestController
 @RequestMapping(value = "/applications", produces = MediaTypes.HAL_JSON_VALUE)
+@Slf4j
 public class ApplicationController {
 
+    private static final String SERVICE_SHORT_NAME = "serviceShortName";
```

Comment:
Make use of pre-defined variables - for reference: [TestConstants.java](https://github.com/UST-MICO/mico/blob/master/mico-core/src/test/java/io/github/ust/mico/core/TestConstants.java)

Reply:
I don't think we should mix test constants and constants in our controller logic.